pages/workspaces/[workspaceId]/activity.tsx

```diff
@@ -153,7 +148,7 @@ const WorkspaceActivityPage = ({
                   selectedTab={"pull requests"}
                   pageId={`/workspaces/${workspace.id}`}
                 />
-                {isMobile ? <DayRangePicker /> : null}
+                <ClientOnly>{isMobile ? <DayRangePicker /> : null}</ClientOnly>
```

Comment:
Hydration error fixes. These were preventing me from running the changes to E2E tests.